src/global_mapper/global_mapper.h

```diff
@@ -6,16 +6,22 @@
 #include <vector>
 #include <thread>
 #include <mutex>
+#include <memory>
+#include <condition_variable>
 
 #include "pcl_ros/point_cloud.h"
+#include "global_mapper/global_mapper_params.h"
+
+#include "occ_map/voxel_map.hpp"
+#include "occ_map/pixel_map.hpp"
 
 namespace global_mapper {
 
 typedef pcl::PointCloud<pcl::PointXYZ> PointCloud;
```

Comment:
Good link for those interested: http://stackoverflow.com/questions/24916435/why-do-type-aliases-in-c-use-using-instead-of-typedef-in-their-syntax